entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -eo pipefail
+
+LOCUST_BIN='/usr/bin/locust'
+LOCUST_SCRIPT="${LOCUST_SCRIPT:-/locust-tasks/tasks.py}"
+LOCUST_MODE="${LOCUST_MODE:-standalone}"
+
+TARGET_HOST="${TARGET_HOST:?Required variable not set}"
+
+if [ "$LOCUST_MODE" == "master" ]; then
+  LOCUST_CMD="${LOCUST_BIN} -f ${LOCUST_SCRIPT} --host ${TARGET_HOST} --master"
+elif [[ "$LOCUST_MODE" = "worker" ]]; then
```

Comment:
Why does this branch use double brackets and single equals while the previous branch uses single brackets and double equals?

Reply:
Good point, thanks for noticing this!